packages/sitecore-jss-angular/src/components/date.directive.ts

```diff
@@ -62,11 +58,14 @@ export class DateDirective extends BaseFieldDirective implements OnChanges {
       return;
     }
 
+    this.renderMetadataTag('open');
```

Comment:
I'm thinking if it looks a bit confusing, since at the first glance it means that we render metadata tag in any case, while inside of the function we have a check for "metadata" field. @stasmaxymov any thoughts?

Reply:
@illiakovalenko  I didn't want to surface any checks to dirived fields, to not further complicate it. or risk that in customer field implementation it will ignored. As a simple mitigation I can suggest renaming to `renderMetadata`, without `Tag` suffix it doesn't create unnecessary expectation on what will be rendered (or not redered)

Reply:
Makes sense to me, we can name it as "renderMetadata"

Reply:
done